openshift/patches/0006-tekton-pipelines.patch

```diff
@@ -32,3 +32,21 @@ index bf90adfa..4e655633 100644
  metadata:
    name: func-s2i
    labels:
+@@ -26,6 +26,7 @@ spec:
+       description: Reference of the image S2I will produce.
+     - name: REGISTRY
+       description: The registry associated with the function image.
++      default: ""
+     - name: PATH_CONTEXT
+       description: The location of the path to run s2i from.
+       default: .
+@@ -114,6 +115,9 @@ spec:
+           $(params.IMAGE) docker://$(params.IMAGE)
+
+         cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST
++        echo ""
```

Comment:
what is this for?

Reply:
isn't that just debugging?

Reply:
That file is used by the pipeline, iiuc, here: https://github.com/openshift-knative/kn-plugin-func/blob/a80fa69fa32b6c42575b59629332c74ede8f77c0/openshift/overrides/pkg/pipelines/resources/tekton/pipeline/dev-console/0.1/nodejs-pipeline.yaml#L82

Reply:
I do understand `cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST` but what about the three lines below?

Reply:
iiuic `cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST` also prints the hash, so `cat $(workspaces.source.path)/image-digest` seems redundant, am I wrong?

Reply:
No, you are not wrong. https://github.com/openshift-knative/kn-plugin-func/pull/573